docs/egress-ip.md

```diff
@@ -115,27 +115,28 @@ COMMIT
 ...
 ```
 
-IPRoute2 rules will look like the following - note rule with priority `6000` and also the refered table `1008`:
+IPRoute2 rules will look like the following - note rule with priority `6000` and also the table `1111`:
 ```shell
 sh-5.2# ip rule
 0:	from all lookup local
 30:	from all fwmark 0x1745ec lookup 7
-6000:	from 10.244.2.3 lookup 1008
+6000:	from 10.244.2.3 lookup 1111
 32766:	from all lookup main
 32767:	from all lookup default
 ```
 
-And the default route in the correct table `1008`:
+And the default route in the correct table `1111`:
 ```shell
-sh-5.2# ip route show table 1008
+sh-5.2# ip route show table 1111
 default dev dummy
 ```
 
 No NAT is required on the OVN managed primary network gateway router.
 OVN-Kubernetes (ovnkube-node) takes care of adding a rule to the rule table with src IP of the pod and routed towards a
-new routing table specifically created to route the traffic out the correct interface. IPTables is also altered and an entry
-is created within the chain `OVN-KUBE-EGRESS-IP-Multi-NIC` for each pod to allow SNAT to occur when a src IP is match
-leaving a particular interface.
+new routing table specifically created to route the traffic out the correct interface. IPTables are also altered and an entry
+is created within the chain `OVN-KUBE-EGRESS-IP-Multi-NIC` for each selected pod to allow SNAT to occur when
+egress-ing a particular interface. The routing table number `1111` is generated from the interface name.
+Routes within the main table which egress the interface picked to host the Egress IP are also added to this routing table.
```

Comment:
Routes within the main routing table who's output interface share the same interface used for Egress IP are also cloned into the VRF 1111.